digicamviewer/viewer.py

```diff
@@ -119,11 +118,12 @@ def next(self, event=None, step=1):
         for i, event in zip(range(step), self.event_stream):
             pass
 
+        telescope_id = event.r0.tels_with_data[0]
```

Comment:
Fine for me but if you want to display telescope number x you can't.

Reply:
yes correct